content/outcomes/secure-software-supply-chain/establish-capability/establish-devsecops.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Developing a DevSecOps Practice
+weight: 1
+layout: single
+team:
+- Craig Tracey
+---
+
+Modern software delivery demands that we think differently about how we bring
+code to production. Traditional methodologies typically took on a siloed approach:
+a development team writes the code, a security team ensures that this code
+adheres to organizational compliance requirements, and operations teams
+maintained the availability of the developed service. While this organization of
+specific roles often met the demand for bringing code to production, it suffers
```

Comment:
Something feels off about this sentence - we switch from past tense (often met) to present tense (bringing code to production).